python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -1199,6 +1200,20 @@ def graph_multi_asset(
             group name will be applied to all assets produced by this multi_asset.
         can_subset (bool): Whether this asset's computation can emit a subset of the asset
             keys based on the context.selected_assets argument. Defaults to False.
+        config (Optional[Union[ConfigMapping], Mapping[str, Any]):
+            Describes how the graph underlying the asset is configured at runtime.
+
+            If a :py:class:`ConfigMapping` object is provided, then the graph takes on the config
```

Comment:
[1] should this say "takes on the config of this object" instead of "config schema" i thought "config_schema" was the key:type mapping, not the key:value mapping, but maybe i'm misunderstanding

Reply:
`ConfigMapping` exposes its own `config_schema` so thats whats referred to